### ISSUE TITLE
Upload speed cap

### ISSUE THREAD

**Issue by pargnyce** [jsmsj/sa-drive#2]
The upload speed only ranges up to 70mbps. Is it a normal thing?

**Comment by jsmsj** [comment]
No, there is no upload speed cap as such. (Why would it even be there lol ?)
